replica/database.cc

```diff
@@ -638,6 +638,10 @@ database::setup_metrics() {
 
         sm::make_total_operations("total_view_updates_on_wrong_node", _cf_stats.total_view_updates_on_wrong_node,
                 sm::description("Total number of view updates which are computed on the wrong node.")).set_skip_when_empty(),
+
+        sm::make_total_operations("total_base_view_replicas_mismatch", _cf_stats.total_base_view_replicas_mismatch,
+                sm::description("Total number of view updates for which the base had a different replica count than the view. "
```

Comment:
I have some problems with this description, but maybe it's because I don't understand things enough. This metric is updated only when `get_view_natural_endpoint()` is called, so if nobody calls it, and there's a mismatch between the base and the view, this metric will not be bumped. That's one thing. The other thing is that this mistmach is only temporary, right? Maybe it's worth including both these remarks in the description of this metric.

Reply:
We call `get_view_natural_endpoint()` during every view update, and not problematic if we don't do view updates during the mismatch, so I'm not sure what to add about that.
I added a bit about it being temporary in the rebase.